data-plane-framework/draft-ietf-detnet-data-plane-framework.xml

```diff
@@ -635,6 +657,22 @@ CE1----EN1--------R1-------R2-------R3--------EN2-----CE2
      prevent misordering of packets. In both cases the protection paths are
      established and maintained for the duration of the DetNet service.
      </t>
+   </section>
+     <section title="Path Differential Delay">
+             <t>
+     In the preceding example, reordering of packets is dependent on the number
+     of out-of-order packets that can be buffered and the delay difference of
+     arriving packets.  DetNet uses configuration of the number of maximum
+     number out-of-order packets and the maximum delay that out-of-order
+     packets can be held before being delivered. If the differential delay

```

Comment:
Maximum delay is a parameter for the DetNet flow and not for the delay of "out-of-order" packets. However they are somewhat related. "Maximum delay" decreased by the "actual delay of the packet" gives the delay what can be used during re-ordering.

Reply:
[Don] Please suggest corrected text.

[Balazs] What about something like this:
"DetNet uses for configuration of POF related buffers 
the DetNet flow specific requirements (e.g., maximum
number of out-of-order packets, maximum latency 
of the DetNet flow, etc.)."